src/d3a/d3a_core/sim_results/plotly_graph.py

```diff
@@ -62,7 +59,7 @@ def _invert(inlist: list): return [-1 * ll for ll in inlist]
 
 
 def _get_color(key, alpha):
-    return DEVICE_PLOT_COLORS[key].replace("alpha", str(alpha))
+    return DEVICE_PLOT_COLORS.get(key, green).replace("alpha", str(alpha))
```

Comment:
What's the rational behind falling to this default value ?

Reply:
You can check here: https://github.com/gridsingularity/d3a/pull/1159/files/3b073c1caa090c0499cd74ce34df080d15bc80a1#diff-a112ac89fc68de1a5551c06f65c8d71598bfdf7b440ad5bd883d2242c0d05ceaL37-L44

Before my changes, most of the keys all had the same value. So I just provided that value (`green`) as default and removed those lines.

Reply:
Here for a change I would have voted for the more verbose version (the original version) for readability. But leave it. It is not important part of the code.

Reply:
Yes, the old version was more explicit. It was also more error-prone (if the key didn't exist in the dictionary, it would have raised an exception). But this is not necessarily bad, actually.